src/instr/instr_paje_types.cpp

```diff
@@ -52,7 +52,7 @@ void PJ_type_free (type_t type)
   xbt_dict_cursor_t cursor = nullptr;
   xbt_dict_foreach (type->values, cursor, value_name, val) {
     XBT_DEBUG("free value %s, child of %s", val->name, val->father->name);
-    delete val;
+    free(val);
```

Comment:
Good catch, the value is not created by new and thus should not be freed with delete. I think that the best solution is to actually use new and delete, but this is already an improvement, thanks.